### ISSUE TITLE
Attendee - My Tickets - Fix purchased ticket animation 

### ISSUE THREAD

**Issue by Mcscott142** [revelrylabs/bn-mobile-react#311]
## Background
Through the checkout flow work, we lost the animation for My Tickets when a new ticket is purchased. This ticket is for the animation after purchasing a ticket on the My Tickets screen. Look at 0:45 second mark in this [video](https://vimeo.com/291552558) pass is `t@ri`.

## Scenario: A user selects Purchase Ticket
- [ ] When I Purchase a ticket
- [ ] Then I'm redirected to My Tickets
- [ ] And I see the loading animation for my new ticket (video in description)

**Comment by jwietelmann** [comment]
investigating but not marking in progress until i think i understand it enough to score

**Comment by jwietelmann** [comment]
okay this should be worked after #325 is fixed, IMO. the root of the issue is just that the list isn't re-fetching. if after that the animation doesn't work, we can look at it then.

**Comment by jwietelmann** [comment]
when #340 is merged, this will be ready to work

**Comment by daybreaker** [comment]
Have this working, just trying to clean up some stuff like clearing out the boolean triggering the bounce.
